fmc/src/flow/pcr.rs

```diff
@@ -61,11 +69,63 @@ fn extend_pcr_common(env: &mut FmcEnv, hand_off: &HandOff, pcr_id: PcrId) -> Cal
     let rt_tci: [u8; 48] = okref(&rt_tci)?.into();
     env.pcr_bank.extend_pcr(pcr_id, &mut env.sha384, &rt_tci)?;
 
+    log_pcr(
```

Comment:
Optional but we can be a bit more efficient with the logs by refactoring to emit one log and indicating which PCRs it ends up in based on the reset reason. But I think that would be a pretty invasive change, and it doesn't look like we're logging too much here.

Reply:
Let's defer it to a future PR then.